apps/test/unit/storage/dataBrowser/dataVisualizer/VisualizerModalTest.js

```diff
@@ -143,4 +143,64 @@ describe('VisualizerModal', () => {
       ).to.deep.equal(expectedNumericColumns);
     });
   });
+
+  describe('getValuesForFilterColumn', () => {
```

Comment:
Other test cases I'd consider:

- What do we do with strings containing quotes? (`'"Who's there?" he said.'`)
- Columns contain object or array values (if possible)
  - How deduplication of values works for objects or arrays (if possible)
- Can `NaN` be stored in a column? If so, show that deduplication works (since this is a special case in `Set`, given `NaN !== NaN`).